frontend/src/i18n/locales/de.yaml

```diff
@@ -11,6 +11,7 @@ navigation: Navigation
 rename: Umbenennen
 this-field-is-required: Dieses Feld ist erforderlich.
 copy-to-clipboard: In Zwischenablage kopieren
+version-information: Versionsauskunft
```

Comment:
Uff, that sounds verry german to me :D How about just "Version"? Or even, "Versionsinformationen" as a more direct translation.

Reply:
I mean it **is** the German translation after all. :stuck_out_tongue_winking_eye: But sure, we can just have it say "version". :shrug: